index.html

```diff
@@ -0,0 +1,689 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
```

Comment:
- [ ] `[optional]` it's better to use `./styles.css` on the link here, it works fine now but if the file wasn't in the root directory (for example in a folder called styles) it would break once you deploy it to github pages or anything similar